src/test/java/homePage/HomeTest.java

```diff
@@ -0,0 +1,23 @@
+package homePage;
+import base.BaseTests;
+import base.TestListeners;
+import org.testng.Assert;
+import org.testng.annotations.Listeners;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.RegisterPage;
+
+import static utils.DataDriven.JsonClass.readJson;
+
+@Listeners(TestListeners.class)
+public class HomeTest extends BaseTests {
+
+
+    @Test
+    public void TestHomePageIsVisible(){
+
+        HomePage homePage1 = homePage;
```

Comment:
No need to create a new object; the homepage object defined in the BaseTests class can be used.

Reply:
It is possible to add the verification step of the homepage in the RegisterTests class, as it is considered one of the steps of the test case, instead of creating a separate test case for the verification in a different class